content/open-access.md

```diff
@@ -0,0 +1,55 @@
+# Open Access
+
+Open Access refers to online, free of cost access to research outputs, such as publications, with limited copyright and licensing restrictions so that anyone may read and reuse that research.
+
+Traditionally, success in scholarly research is measured by impact factor - a researcher is often deemed more successful and advances more quickly in their career if they publish in journals with high impact factors (the "publish or perish" mantra in research). However, many of these high impact factor journals are subscription based, meaning that the research is locked behind a paywall with high fees associated with accessing it or high costs to the researcher for making it open access upon publication.
+
+In academia, there is widespread dissatisfaction with this expensive traditional publishing model which is unsustainable and redundant. In the current publishing landscape, Universities, research institutions and funding agencies are often double paying for the research they produce - they have to pay the high journal subscription costs in order to access research articles, for which they have already funded the research and publication of. Meanwhile, journal costs continue to increase while library budgets continue to decrease, and are already unaffordable to smaller research institutes and underrepresented communities.
```

Comment:
Is this paragraph necessary? I like it, but it feels too advocate-y for an information page.